scripts/preprocess.sh

```diff
@@ -39,7 +39,21 @@ if [[ "${CF_INSTANCE_INDEX}" = "0" ]]; then
   # Run updatedb if necessary
   UPDATEDB_STATUS=$(drush updatedb-status 2>/dev/null)
   if [[ $UPDATEDB_STATUS != "" ]]; then
-    drush updatedb --no-cache-clear
+    echo "Updates required"
+    error_file=$(mktemp)
+    UPDATEDB_OUTPUT+=$(drush updatedb -y --no-cache-clear 2>$error_file)
+    err=$(< $error_file)
+    case "${err}" in
+      *"Update failed"*)
+        echo "An error occured"
+        echo $err
+        exit
```

Comment:
This will abort `preprocess.sh` with an exit code of `0` - you probably want to exit with a non-zero code if an error has occurred. But actually I cant remember if a non-zero exit code in `preprocess.sh` will cause the `cf push` to error out - easy enough to test though.

Reply:
I've added an exit code of `1`.